microsoft-edge/hosting/webview2/GettingStarted.md

```diff
@@ -37,11 +37,15 @@ Now let's add the WebView2 SDK into the project. For the developer preview, you
 
 ![manageNugetPackages](images/manageNugetPackages.PNG)
 
-2. Enter **Microsoft.Web.WebView2** in the search bar, click **Microsoft.Web.WebView2** from the results, and click **Install** in the right hand side window and install the latest SDK. Nuget will download the SDK to your machine.
+2. Enter **Microsoft.Windows.ImplementationLibrary** in the search bar, click **Microsoft.Windows.ImplementationLibrary** from the results, and click **Install** in the right hand side window and install the latest SDK. Nuget will download the SDK to your machine.
```

Comment:
Great start @pagoe-msft! Some nitpicks -

1. Can we get another PR to point out WIL isn't a necessity when working with WebView2, but merely a good helper library when working with COM? 

2. What version is your VS? Can we get a screenshot for `images/wil.PNG` that is in similar style with `nuget.PNG`. Would be a bit confusing to see 2 screenshots downloading nuget packages having different UIs.